doc/dev/cephfs-mirroring.rst

```diff
@@ -22,6 +27,31 @@ This feature is tracked here: https://tracker.ceph.com/issues/47034
 .. note:: Synchronizing hardlinks is not supported -- hardlinked files get synchronized
           as separate files.
 
+Creating Users
+--------------
+
+Start by creating a user (on the primary/local cluster) for the mirror daemon. This user
+has restrictive capabilities on the MDS and the OSD::
+
+  $ ceph auth get-or-create client.mirror mon 'allow r' mds 'allow r' osd 'allow rw' mgr 'allow r'
```

Comment:
Why does it need `osd 'allow rw'`?

I would suggest just using `fs authorize` instead.

Reply:
mirror daemons need to create mirroring index object (`cephfs_mirror.<instance_id>`).

Reply:
That's the reason to not use `fs authorize` interface -- we need write permission to the data pool (for creating the index object) but have the filesystem read only.

Reply:
one more thing -- the user created here needs to be passed to the mirror daemon. `fs authorize` interface creates a user and grants permission to access a particular file system.

Reply:
`osd 'allow rw tag cephfs *=*'`

Reply:
`osd 'allow rw tag cephfs meta=*, allow r tag cephfs data=*'`

Reply:
nice to know. thx!

Reply:
> osd 'allow rw tag cephfs meta=*, allow r tag cephfs data=*'

works with `metadata=*`